### ISSUE TITLE
2020-12

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#19]

**Comment by qiaojy19** [comment]
# Dec - 2020

## 12.28 - 01.01 

| Title                                                                                                                                                                  |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Federated Block Coordinate Descent Scheme for Learning Global and Personalized Models](https://arxiv.org/pdf/2012.13900)                                              |
| [FLAME: Differentially Private Federated Learning in the Shuffle Model](https://arxiv.org/pdf/2009.08063)                                                              |
| [FedeRank: User Controlled Feedback withFederatedRecommender Systems](https://arxiv.org/pdf/2012.11328)                                                                |
| [FLTrust: Byzantine-robust Federated Learning via Trust Bootstrapping](https://arxiv.org/pdf/2012.13995)                                                               |
| [Federated Multi-Agent Actor-Critic Learning for Age Sensitive Mobile Edge Computing](https://arxiv.org/pdf/2012.14137)                                                |
| [FedMAX: Mitigating Activation Divergence for Accurate and Communication-Efficient Federated Learning](https://arxiv.org/pdf/2004.03657)                               |
| [Federated Unlearning](https://arxiv.org/pdf/2012.13891)                                                                                                               |
| [Timely Communication in Federated Learning](https://arxiv.org/pdf/2012.15831)                                                                                         |
| [PFL-MoE: Personalized Federated Learning Based on Mixture of Experts](https://arxiv.org/pdf/2012.15589)                                                               |
| [Bayesian Federated Learning over Wireless Networks](https://arxiv.org/pdf/2012.15486)                                                                                 |
| [Hybrid FL: Algorithms and Implementation](https://arxiv.org/pdf/2012.12420)                                                                                           |
| [Privacy-preserving Decentralized Aggregation for Federated Learning](https://arxiv.org/pdf/2012.07183)                                                                |
| [Straggler-Resilient Federated Learning: Leveraging the Interplay Between Statistical Accuracy and System Heterogeneity](https://arxiv.org/pdf/2012.14453)             |
| [Dynamic Federated Learning-Based Economic Framework for Internet-of-Vehicles](https://arxiv.org/pdf/2101.00191)                                                       |
| [Fidel: Reconstructing Private Training Samples from Weight Updates in Federated Learning](https://arxiv.org/pdf/2101.00159)                                           |
| [Federated Nonconvex Sparse Learning](https://arxiv.org/pdf/2101.00052)                                                                                                |
| [A Review of Machine Learning Techniques for Applied Eye Fundus and Tongue Digital Image Processing with Diabetes Management System](https://arxiv.org/pdf/2012.15025) |
## 12.21 - 12.25


| Title                                                                                                                                                                      |
| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Toward Understanding the Influence of Individual Clients in Federated Learning](https://arxiv.org/pdf/2012.10936)                                                         |
| [To Talk or to Work: Flexible Communication Compression for Energy Efficient Federated Learning over Heterogeneous Mobile Edge Devices](https://arxiv.org/pdf/2012.11804)  |
| [How to Put Users in Control of their Data in Federated Top-N Recommendation with Learning to Rank](https://arxiv.org/pdf/2008.07192)                                      |
| [Local Differential Privacy based Federated Learning for Internet of Things](https://arxiv.org/pdf/2004.08856)                                                             |
| [Distributed Non-Convex Optimization with Sublinear Speedup under Intermittent Client Availability](https://arxiv.org/pdf/2002.07399)                                      |
| [Communication-Computation Efficient Secure Aggregation for Federated Learning](https://arxiv.org/pdf/2012.05433)                                                          |
| [To Talk or to Work: Energy Efficient Federated Learning over Mobile Devices via the Weight Quantization and 5G Transmission Co-Design](https://arxiv.org/pdf/2012.11070)  |
| [State-of-the-art Techniques in Deep Edge Intelligence](https://arxiv.org/pdf/2008.00824)                                                                                  |
| [Decentralized Federated Learning via Mutual Knowledge Transfer](https://arxiv.org/pdf/2012.13063)                                                                         |
| [Comparison of Privacy-Preserving Distributed Deep Learning Methods in Healthcare](https://arxiv.org/pdf/2012.12591)                                                       |
| [FedEmail: Federated learning in phishing email detection to enabling multiple organizational collaborations without sharing email data](https://arxiv.org/pdf/2007.13300) |
| [Hybrid FL: Algorithms and Implementation](https://arxiv.org/pdf/2012.12420)                                                                                               |
| [Turn Signal Prediction: A Federated Learning Case Study](https://arxiv.org/pdf/2012.12401)                                                                                |
| [Effective Federated Adaptive Gradient Methods with Non-IID Decentralized Data](https://arxiv.org/pdf/2009.06557)                                                          |
| [Coded Federated Learning](https://arxiv.org/pdf/2002.09574)                                                                                                               |

## 12.14 - 12.18

| Title                                                                                                                                                     |
| --------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Addressing Class Imbalance in FederatedLearning](https://arxiv.org/pdf/2008.06217)                                                                       |
| [Model-sharing Games: Analyzing Federated Learning Under Voluntary Participation](https://arxiv.org/pdf/2010.00753)                                       |
| [Cost-Effective Federated Learning Design](https://arxiv.org/pdf/2012.08336)                                                                              |
| [Federated Mimic Learning for Privacy Preserving Intrusion Detection](https://arxiv.org/pdf/2012.06974)                                                   |
| [Federated Transfer Learning for EEG Signal Classification](https://arxiv.org/pdf/2004.12321)                                                             |
| [Backdoor Attacks on Federated Meta-Learning](https://arxiv.org/pdf/2006.07026)                                                                           |
| [Privacy Preserving Demand Forecasting to Encourage Consumer Acceptance of Smart Energy Meters](https://arxiv.org/pdf/2012.07449)                         |
| [FedHome: Cloud-Edge based Personalized Federated Learning for In-Home Health Monitoring](https://arxiv.org/pdf/2012.07450)                               |
| [FedADC: Accelerated Federated Learning with Drift Control](https://arxiv.org/pdf/2012.09102)                                                             |
| [More Industry-friendly: Federated Learning with High Efficient Design](https://arxiv.org/pdf/2012.08809)                                                 |
| [Inexact-ADMM based Federated Meta-Learning for Fast and Continual Edge Learning](https://arxiv.org/pdf/2012.08677)                                       |
| [Personalized Federated Learning with First Order Model Optimization](https://arxiv.org/pdf/2012.08565)                                                   |
| [Beyond Privacy Trade-offs with Structured Transparency](https://arxiv.org/pdf/2012.08347)                                                                |
| [CosSGD: Nonlinear Quantization for Communication-efficient Federated Learning](https://arxiv.org/pdf/2012.08241)                                         |
| [Towards open and expandable cognitive AI architectures for large-scale multi-agent human-robot collaborative learning](https://arxiv.org/pdf/2012.08174) |
| [Bandit-based Communication-Efficient Client Selection Strategies for FederatedLearning](https://arxiv.org/pdf/2012.08009)                                |
| [UVeQFed: Universal Vector Quantization for Federated Learning](https://arxiv.org/pdf/2006.03262)                                                         |
| [Federated Learning under Importance Sampling](https://arxiv.org/pdf/2012.07383)                                                                          |
| [Privacy-preserving Decentralized Federated Learning](https://arxiv.org/pdf/2012.07183)                                                                   |
| [Social Distancing and the Internet: What Can Network Performance Measurements Tell Us?](https://arxiv.org/pdf/2012.09850)                                |
| [Learning from History for Byzantine Robust Optimization](https://arxiv.org/pdf/2012.10333)                                                               |
| [Fairness and Accuracy in Federated Learning](https://arxiv.org/pdf/2012.10069)                                                                           |
| [Federated Learning with Unbiased Gradient Aggregation and Controllable Meta Updating](https://arxiv.org/pdf/1910.08234)                                  |
## 12.07 - 12.11

| Title                                                                                                                                                              |
| ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| [Design and Analysis of Uplink and Downlink Communications for Federated Learning](https://arxiv.org/pdf/2012.04057)                                               |
| [Multi-site fMRI Analysis Using Privacy-preserving Federated Learning and Domain Adaptation: ABIDE Results](https://arxiv.org/pdf/2001.05647)                      |
| [Can Federated Learning Save The Planet ?](https://arxiv.org/pdf/2010.06537)                                                                                       |
| [Confederated Machine Learning on Horizontally and Vertically Separated Medical Data for Large-Scale Health System Intelligence](https://arxiv.org/pdf/1910.02109) |
| [Spiking Neural Networks -- Part III: Neuromorphic Communications](https://arxiv.org/pdf/2010.14220)                                                               |
| [Federated Learning in Unreliable and Resource-Constrained Cellular Wireless Networks](https://arxiv.org/pdf/2012.05137)                                           |
| [Accurate and Fast Federated Learning via IID and Communication-Aware Grouping](https://arxiv.org/pdf/2012.04857)                                                  |
| [Reconfigurable Intelligent Surface Enabled Federated Learning: A Unified Communication-Learning Design Approach](https://arxiv.org/pdf/2011.10282)                |
| [Towards Communication-efficient and Attack-Resistant Federated Edge Learning for Industrial Internet of Things](https://arxiv.org/pdf/2012.04436)                 |
| [RC-SSFL: Towards Robust and Communication-efficient Semi-supervised Federated Learning System](https://arxiv.org/pdf/2012.04432)                                  |
| [Blockchain-Federated-Learning and Deep Learning Models for COVID-19 detection using CT Imaging](https://arxiv.org/pdf/2007.06537)                                 |
| [Federated Multi-Task Learning for Competing Constraints](https://arxiv.org/pdf/2012.04221)                                                                        |
| [GraphFL: A Federated Learning Framework for Semi-Supervised Node Classification on Graphs](https://arxiv.org/pdf/2012.04187)                                      |
| [Improved Convergence Rates for Non-Convex Federated Learning with Compression](https://arxiv.org/pdf/2012.04061)                                                  |
| [Dynamic Clustering in Federated Learning](https://arxiv.org/pdf/2012.03788)                                                                                       |
| [Vehicular Cooperative Perception Through Action Branching and Federated Reinforcement Learning](https://arxiv.org/pdf/2012.03414)                                 |
| [SoK: Training Machine Learning Models over Multiple Sources with Privacy Preservation](https://arxiv.org/pdf/2012.03386)                                          |
| [FedSemi: An Adaptive Federated Semi-Supervised Learning Framework](https://arxiv.org/pdf/2012.03292)                                                              |
| [Accurate and Fast Federated Learning via Combinatorial Multi-Armed Bandits](https://arxiv.org/pdf/2012.03270)                                                     |
| [TornadoAggregate: Accurate and Scalable Federated Learning via the Ring-Based Architecture](https://arxiv.org/pdf/2012.03214)                                     |
| [DONE: Distributed Newton-type Method for Federated Edge Learning](https://arxiv.org/pdf/2012.05625)                                                               |
| [Analysis and Optimal Edge Assignment For Hierarchical Federated Learning on Non-IID Data](https://arxiv.org/pdf/2012.05622)                                       |
| [Communication-Computation Efficient Secure Aggregation for Federated Learning](https://arxiv.org/pdf/2012.05433)                                                  |
| [Privacy Amplification by Decentralization](https://arxiv.org/pdf/2012.05326)                                                                                      |



## 11.30 - 12.04


| Title                                                                                                                                                         |
| ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Personalized Cross-Silo Federated Learning on Non-IID Data](https://arxiv.org/pdf/2007.03797.pdf)                                                            |
| [Automated Metadata Harmonization Using Entity Resolution & Contextual Embedding](https://arxiv.org/pdf/2010.11827.pdf)                                       |
| [MYSTIKO : : Cloud-Mediated, Private, Federated Gradient Descent](https://arxiv.org/pdf/2012.00740.pdf)                                                       |
| [FLeet: Online Federated Learning via Staleness Awareness and Performance Prediction](https://arxiv.org/pdf/2006.07273.pdf)                                   |
| [Federated Learning with Heterogeneous Labels and Models for Mobile Activity Monitoring](https://arxiv.org/pdf/2012.02539.pdf)                                |
| [FAT: Federated Adversarial Training](https://arxiv.org/pdf/2012.01791.pdf)                                                                                   |
| [Asynchronous Federated Optimization](https://arxiv.org/pdf/1903.03934.pdf)                                                                                   |
| [One-Bit Over-the-Air Aggregation for Communication-Efficient Federated Edge Learning: Design and Convergence Analysis](https://arxiv.org/pdf/2001.05713.pdf) |
| [A Systematic Literature Review on Federated Learning: From A Model Quality Perspective](https://arxiv.org/pdf/2012.01973)                                    |
| [A Systematic Literature Review on Federated Machine Learning: From A Software Engineering Perspective](https://arxiv.org/pdf/2007.11354.pdf)                 |
| [Adaptive Federated Optimization](https://arxiv.org/pdf/2003.00295)                                                                                           |
| [Blockchain Assisted Decentralized Federated Learning (BLADE-FL) with Lazy Clients](https://arxiv.org/pdf/2012.02044.pdf)                                     |
| [Communication-Efficient Federated Distillation](https://arxiv.org/pdf/2012.00632.pdf)                                                                        |
| [Distributed Machine Learning for Wireless Communication Networks: Techniques, Architectures, and Applications](https://arxiv.org/pdf/2012.01489.pdf)         |
| [Edge-assisted Democratized Learning Towards Federated Analytics](https://arxiv.org/pdf/2012.00425)                                                           |
| [Fast-Convergent Federated Learning with Adaptive Weighting](https://arxiv.org/pdf/2012.00661.pdf)                                                            |
| [Federated Learning with Diversified Preference for Humor Recognition](https://arxiv.org/pdf/2012.01675.pdf)                                                  |
| [Federated Marginal Personalization for ASR Rescoring](https://arxiv.org/pdf/2012.00898.pdf)                                                                  |
| [Mitigating Bias in Federated Learning](https://arxiv.org/pdf/2012.02447.pdf)                                                                                 |
| [Robust Federated Learning with Noisy Labels](https://arxiv.org/pdf/2012.01700.pdf)                                                                           |
| [Second-Order Guarantees in Federated Learning](https://arxiv.org/pdf/2012.01474.pdf)                                                                         |
| [Toward Robustness and Privacy in Federated Learning: Experimenting with Local and Central Differential Privacy](https://arxiv.org/pdf/2009.03561.pdf)        |
